spec/models/comfy/cms/page_spec.rb

```diff
@@ -32,6 +84,21 @@
       end
     end
 
+    context 'ignores suppressed articles' do
+      let(:animals_tag) { Tag.create value: 'animals' }
+      let!(:animals_article) { create :english_article, keywords: [animals_tag] }
+
+      it 'order by most matching tags' do
+        create :english_article, keywords: [animals_tag], suppress_from_links_recirculation: true
+        not_suppressed_match_article = create :english_article,
```

Comment:
Sure. The intention was to keep the important parts ( the setting of suppress_from_links_recirculation ) local/close to the test for clarity.